SDL/SDLJoystick.cpp

```diff
@@ -186,7 +189,7 @@ void SDLJoystick::ProcessInput(SDL_Event &event){
 		axis.value = event.caxis.value / 32767.0f;
 		if (axis.value > 1.0f) axis.value = 1.0f;
 		if (axis.value < -1.0f) axis.value = -1.0f;
-		axis.deviceId = DEVICE_ID_PAD_0 + getDeviceIndex(event.caxis.which);
```

Comment:
So for anyone who is not using your new parameter, and who has multiple controller devices, such as probably the author of #8993, this will break their use of PPSSPP.

Wouldn't it be better if your use case was only activated when your parameter was passed?

-[Unknown]